changelogs/fragments/docker-py-changes-1.yml

```diff
@@ -0,0 +1,11 @@
+bugfixes:
+  - "modules and plugins communicating directly with the Docker daemon - fix parsing of IPv6 addresses with a port in ``docker_host``. This is only a change relatively to older community.docker 3.0.0 pre-releases, or with respect to Docker SDK for Python < 6.0.0. Docker SDK for Python 6.0.0 will also include this change (https://github.com/ansible-collections/community.docker/pull/434)."
+  - "modules and plugins communicating directly with the Docker daemon - fix ``ProxyCommand`` handling for SSH connections when not using ``use_ssh_client=true``. This is only a change relatively to older community.docker 3.0.0 pre-releases, or with respect to Docker SDK for Python < 6.0.0. Docker SDK for Python 6.0.0 will also include this change (https://github.com/ansible-collections/community.docker/pull/434)."
+  - "modules and plugins communicating directly with the Docker daemon - do not create a subshell for SSH connections when using ``use_ssh_client=true``. This is only a change relatively to older community.docker 3.0.0 pre-releases, or with respect to Docker SDK for Python < 6.0.0. Docker SDK for Python 6.0.0 will also include this change (https://github.com/ansible-collections/community.docker/pull/434)."
+  - "docker_image - when composing the build context, trim trailing whitespace from ``.dockerignore`` entries. This is only a change relatively to older community.docker 3.0.0 pre-releases, or with respect to Docker SDK for Python < 6.0.0. Docker SDK for Python 6.0.0 will also include this change (https://github.com/ansible-collections/community.docker/pull/434)."
+minor_changes:
+  - "modules and plugins communicating directly with the Docker daemon - improve default TLS version selection for Python 3.6 and newer. This is only a change relatively to older community.docker 3.0.0 pre-releases, or with respect to Docker SDK for Python < 6.0.0. Docker SDK for Python 6.0.0 will also include this change (https://github.com/ansible-collections/community.docker/pull/434)."
+security_fixes:
+  - "modules and plugins communicating directly with the Docker daemon - when connecting by SSH and not using ``use_ssh_client=true``, reject unknown host keys instead of accepting them. This is only a change relatively to older community.docker 3.0.0 pre-releases, or with respect to Docker SDK for Python < 6.0.0. Docker SDK for Python 6.0.0 will also include this change (https://github.com/ansible-collections/community.docker/pull/434)."
+breaking_changes:
+  - "modules and plugins communicating directly with the Docker daemon - when connecting by SSH and not using ``use_ssh_client=true``, reject unknown host keys instead of accepting them. This is only a breaking change relatively to older community.docker 3.0.0 pre-releases, or with respect to Docker SDK for Python < 6.0.0. Docker SDK for Python 6.0.0 will also include this change (https://github.com/ansible-collections/community.docker/pull/434)."
```

Comment:
Nitpick: `s/relatively/relative/'. I like adverbs (my name is Max**well**, after all), but this is not the correct place to use them. Perhaps it also makes sense to shorten `modules and plugins communicating directly with the Docker daemon` to just  `plugins communicating directly with the Docker daemon`, as modules are a type of plugin, but meh.

Reply:
I changed everything except the "modules and plugins ...". That because a) that formulation has been used in earlier changelog fragments, and b) many users do not know that modules are a special kind of plugin. Since the changelog is mainly for users, I think it's important to also mention modules here.